scripts/xsslint/xsslint/reporting.py

```diff
@@ -238,13 +237,16 @@ class SummaryResults(object):
     Contains the summary results for all violations.
     """
 
-    def __init__(self):
+    def __init__(self, ruleset):
         """
         Init method.
+
+        Arguments:
+            ruleset: A RuleSet object containing all of the possible Rules.
         """
         self.total_violations = 0
         self.totals_by_rule = dict.fromkeys(
-            [rule.rule_id for rule in Rules.__members__.values()], 0
+            [rule.rule_id for rule in ruleset.ruleset.values()], 0
```

Comment:
Consider renaming internal set to `rules` or `set` or something that doesn't lead to code like `ruleset.ruleset` which I find confusing to read.  Something like `ruleset.rules` or `ruleset.set`.

This is just something to consider.

Reply:
ok, I'll change to `rules`.

Reply:
Changed.